enterprise/j2ee.api.ejbmodule/nbproject/project.xml

```diff
@@ -228,6 +228,7 @@
             <friend-packages>
                 <friend>com.sun.eview.project</friend>
                 <friend>org.codehaus.mevenide.netbeans.j2ee</friend>
+                <friend>org.netbeans.modules.gradle.javaee</friend>
```

Comment:
It would be better to open to up these modules to general use and just state that they are "under development".

Reply:
That would be certainly welcome, though that may be a different issue and a different PR.